### ISSUE TITLE
MIDI export

### ISSUE THREAD

**Issue by ddelabru** [EntrianAudio/Rack#88]
I'm not sure how this compares in implementation complexity to MIDI import, which is already available. I like Entrian Timeline because a lot of what I do on a piano roll I only plan to use within VCV Rack anyhow.

I have used the MIDI import features a few times, and for some reason assumed there was some kind of corresponding export feature, but just now someone asked if I could share a MIDI of the arrangement I did for an old traditional tune in VCV Rack and I couldn't find the option to export from Timeline. I can convert the module to the free, play-only version, and then if I also replaced or removed any non-Entrian commercial modules I could share the whole patch with Rack users who might not have them, but I think the person who inquired about the MIDI isn't a Rack user at all.

Not an essential feature for me, but I'd be less tempted to do piano roll in a different application sometimes if I knew I could export MIDI from Timeline.

As a user I'm not too worried about MIDI file details like what MIDI instruments are assigned to each channel etc. I figure if I'm exporting and sharing a MIDI I won't mind the extra step of quickly editing those things in another application.

**Comment by EntrianAudio** [comment]
You're right that MIDI export would be a good feature.  In the meantime, you can copy/paste a melody clip into Squinky's Seq++ module, which does have MIDI export.  (I don't know how well that works with complex clips.)

**Comment by brbbbq** [comment]
Do you plan on adding this feature? It seems like MIDI export would be a basic function that would be useful to include. I'm using Entrain to record MIDI sequences from random sequence generators, and I'd like a way to easily export the sequences to use in other projects.

**Comment by roaratorio** [comment]
A MIDI export feature would be incredibly useful. The Entrian modules to a great job of capturing clean MIDI. They could be a fantastic and much needed bridge between the VCV ecosystem and other tools with the addition of an export option.

**Comment by RichieHindle** [comment]
Implemented in Entrian Sequencers 2.3.2.  See the "MIDI export" section in [the manual](https://entrian.com/audio/entrian-sequencers.html).  Thanks for the suggestion!
